.travis.yml

```diff
@@ -80,31 +101,48 @@ before_script:
   - ansible-galaxy collection build .
   - ansible-galaxy collection install $COLLECTION_NAMESPACE-$COLLECTION_NAME-$COLLECTION_VERSION.tar.gz -p /home/travis/.ansible/collections
 
+  # Run all further tests from within the installed directory
+  # Required to resolve imports of other collections
+  - cd /home/travis/.ansible/collections/ansible_collections/$COLLECTION_NAMESPACE/$COLLECTION_NAME
+
+  # Set runme.sh execute permissions stripped by ansible-galaxy. Should be fixed in Ansible 2.10
+  # https://github.com/ansible/ansible/issues/68415
+  - chmod +x tests/integration/targets/inventory/runme.sh
+  - chmod +x tests/integration/targets/inventory/compare_inventory_json.py
+
 script:
-  # Perform unit tests on collection from within the installed directory, not the source directory
-  # Required for imports of other collections (ie. ansible.netcommon) to work correctly
-  - (cd /home/travis/.ansible/collections/ansible_collections/$COLLECTION_NAMESPACE/$COLLECTION_NAME && ansible-test units --coverage --python $PYTHON_VER -v)
 
   # Check python syntax
   - black . --check --diff
 
+  # Sanity tests
+  # --requirements - install pip packages as necessary
+  # Skip pep8 as we use black instead
+  # Skip validate-modules until the issues can be resolved #180
+  - ansible-test sanity -v --requirements --python $PYTHON_VER --skip-test pep8 --skip-test validate-modules
+
+  # Unit tests, with code coverage
+  - ansible-test units -vvvv --coverage --python $PYTHON_VER
```

Comment:
After looking at the tests, what is the least amount of verbosity can we get away with these commands? The test output is pretty heavy with setup information for all this.

Reply:
Yeah it's very verbose. I wasn't sure whether to go for enough information to debug any issues, or just enough to read and see where it failed. These could probably all be changed to just `-v` or `-vv`

Reply:
Yes that is definitely something to consider. I'll leave that decision up to you.

Reply:
I've changed it to `-v`. Should be enough for most cases.